engine/consensus/dkg/reactor_engine_test.go

```diff
@@ -449,22 +465,22 @@ func (suite *ReactorEngineSuite_CommittedPhase) TestStartupInCommittedPhase_DKGS
 		mock.Anything,
 		mock.Anything,
 	)
-	// should set DKG end state
-	suite.Assert().Equal(flow.RandomBeaconKeyCommitted, suite.DKGState)
+	// ensure we commit my beacon private key
+	suite.dkgState.AssertCalled(suite.T(), "CommitMyBeaconPrivateKey", suite.NextEpochCounter(), entry.NextEpochCommit)
 }
 
 // TestStartupInCommittedPhase_DKGSuccess tests that the dkg end state is correctly
 // set when starting in EpochCommitted phase and the DKG end state is already set.
 func (suite *ReactorEngineSuite_CommittedPhase) TestStartupInCommittedPhase_DKGStateAlreadySet() {
 
-	// we are in the EpochSetup phase
+	// we are in the Epoch Commit phase
 	suite.snap.On("EpochPhase").Return(flow.EpochPhaseCommitted, nil).Once()
 	// the dkg for this epoch has been started and ended
 	suite.dkgState.On("IsDKGStarted", suite.NextEpochCounter()).Return(true, nil).Once()
-	suite.dkgState.On("GetDKGState", suite.NextEpochCounter()).Return(flow.DKGStateFailure, nil).Once()
+	suite.DKGState = flow.DKGStateFailure
 
 	// start up the engine
-	unittest.AssertClosesBefore(suite.T(), suite.engine.Ready(), time.Second)
+	unittest.AssertClosesBefore(suite.T(), suite.engine.Ready(), 100*time.Second)
```

Comment:
Is it really not starting up within a second? 100s seems like a lot.. Maybe 5s or 10s?